privacy_evaluator/models/torch/_train_dcti.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+
+import torch
+import torch.nn as nn
+import torch.optim as optim
+
+import torchvision
+import torchvision.datasets as datasets
+import torchvision.transforms as transforms
+
+from torch.utils.data import DataLoader
```

Comment:
Ah and would be very nice if you could add the dependencies to the requirements.txt file